examples/ping-probe/probe.js

```diff
@@ -0,0 +1,81 @@
+// Configuration of InfluxDB writer
+const INFLUXDB_API_KEY = "FILL ME IN!";
+const INFLUXDB_ORG_ID = "FILL ME IN!";
+const INFLUXDB_BUCKET = "FILL ME IN!";
+const INFLUXDB_ENDPOINT = "https://eu-central-1-1.aws.cloud2.influxdata.com/";
```

Comment:
Will this be the same for all users, or should we parameterise it?

Reply:
Good point, it will change

Reply:
It will change. I am keeping this value to show what the correct value looks like - it's the "base path" without any specific endpoint path.